### ISSUE TITLE
nof5 requires vm-browserify on windows

### ISSUE THREAD

**Issue by jhnns** [peerigon/nof5#8]
I have totally no idea whats going on here, but when I try to use nof5 on windows `/AppData/Roaming/npm/node_modules/nof5/node_modules/browserify/node_modules/vm-browserify/index.js` appears in the scripts view. This module doesn't appear when I run nof5 on linux.

**Comment by jhnns** [comment]
This is also the cause for the rewire problem I've told you.

**Comment by topa** [comment]
Fist I need to install a Windows-VM to reproduce the described error ...

**Comment by matthaias** [comment]
It seems that the dependency rewire is missing, but it shouldn't because nof5 should be independent from rewire.

**Comment by jhnns** [comment]
Yes, that's what it seems. But the error is caused by the problem as described above. I know why `rewire` seems like causing the problem (it's related to how the rewire browserify middleware works).

If we solve the problem that `vm-browserify` isn't required anymore, the `rewire` error will disappear too.
